### PR TITLE
Add section about office hours to TP page

### PR DESCRIPTION
**Pull request by lukassim**
## Description

Add a section and some details about office hours for translators to the translation program page.

### CODE REVIEW

**Review by wackerow — APPROVED** [review]

File: src/content/contributing/translation-program/index.md [hunk 1]
```diff
@@ -102,6 +102,16 @@ You can also reach out to us at translations@ethereum.org
 
 Thank you for your participation in the ethereum.org Translation Program!
 
+**Office hours for translators**
+
+We will be holding office hours for translators on the second Wednesday of every month. These will be held in the #office-hours voice channel on our Discord, where you can also find the exact times and additional details.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:107]:
```suggestion
We have office hours for translators on the second Wednesday of every month. These are held in the #office-hours voice channel on the [ethereum.org Discord](https://discord.gg/geKhWjtF), where you can also find the exact times and additional details.
```

File: src/content/contributing/translation-program/index.md [hunk 2]
```diff
@@ -102,6 +102,16 @@ You can also reach out to us at translations@ethereum.org
...
+
+The purpose of the office hours is to allow our translators to ask questions about the translation process, provide feedback on the program, share their ideas, or just have a chat with us.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:109]:
```suggestion
The purpose of office hours is to allow our translators to ask questions about the translation process, provide feedback on the program, share their ideas, or just have a chat with us.
```

File: src/content/contributing/translation-program/index.md [hunk 3]
```diff
@@ -102,6 +102,16 @@ You can also reach out to us at translations@ethereum.org
...
+Participants can also learn more about the translation program, and get some information on how to become an ethereum.org translator.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:110]:
```suggestion
Participants can also learn more about the translation program and get some information on becoming an ethereum.org translator.
```

File: src/content/contributing/translation-program/index.md [hunk 4]
```diff
@@ -102,6 +102,16 @@ You can also reach out to us at translations@ethereum.org
...
+Finally, we want to use these calls to communicate recent developments in connection with the translation program, and share key tips and instructions with our contributors.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:111]:
```suggestion
Finally, we want to use these calls to communicate recent developments with the translation program and share key tips and instructions with our contributors.
```

File: src/content/contributing/translation-program/index.md [hunk 5]
```diff
@@ -102,6 +102,16 @@ You can also reach out to us at translations@ethereum.org
...
+
+If you are an ethereum.org translator, or would like to become one, feel free to join us during one of these sessions.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:113]:
```suggestion
If you are an ethereum.org translator or would like to become one, feel free to join us during one of these sessions.
```

**Review by minimalsm — CHANGES_REQUESTED** [review]

**Comment by minimalsm** [comment]
🚀🚀🚀

**Pull request merged**